ts-cloudfs/index.ts

```diff
@@ -1,11 +1,11 @@
 /**
- * @topology_group main
+ * @klotho::execution_unit {
+ *  name = "cloudfs-main"
+ * }
  */
 
 
-/**
- * @capability file_persist
- */
+// @klotho::persist
```

Comment:
will need to update these w/ the types "kv, fs, secret"

Reply:
I plan to add the support for this soon.